tdrs-frontend/src/selectors/auth.js

```diff
@@ -16,7 +35,9 @@ export const accountStatussIsPending = (state) =>
   selectUserAccountApprovalStatus(state) === 'Pending'
 
 export const accountIsInReview = (state) =>
-  accountStatusIsAccessRequestSubmitted(state) || accountStatussIsPending(state)
+  accountStatusIsAccessRequestSubmitted(state) ||
+  accountStatussIsPending(state) ||
+  (accountStatusIsApproved(state) && selectUserRoles(state).length === 0)
```

Comment:
the actual bug fix is here. treat `Approved` users with no role as being "Under review"